modular_skyrat/code/datums/speech_mod/impediments.dm

```diff
@@ -28,3 +28,9 @@
 	speech_args[SPEECH_MESSAGE] = replacetext(speech_args[SPEECH_MESSAGE], "l", "w")
 	speech_args[SPEECH_MESSAGE] = replacetext(speech_args[SPEECH_MESSAGE], "L", "W")
 
+/datum/speech_mod/impediment_cheems
```

Comment:
```suggestion
/datum/speech_mod/impediment_nm
```

Reply:
We have a pretty well-done speech impediment system right now. We really just need to add some way to implement speech quirks in a less... clusterfuck way.